my-skill/scripts/validate.js

```
@@ -0,0 +1,195 @@
+
+import { readFileSync, existsSync } from 'fs'
+import { resolve, dirname } from 'path'
+import { fileURLToPath } from 'url'
+
+const __dirname = dirname(fileURLToPath(import.meta.url))
+const ROOT = resolve(__dirname, '../../')
+
+let failures = 0
+
+function pass(msg) {
+  console.log(`  ✓  ${msg}`)
```

ESLint

[error] 'console' is not defined.

(no-undef)